aptos-move/framework/aptos-framework/sources/apt_primary_fungible_store.move

```diff
@@ -0,0 +1,77 @@
+
+module aptos_framework::apt_primary_fungible_store {
```

Comment:
Might have forgotten some context here - why do we need a standalone module? With the recent dispatchable fungible asset, I think our API/interface is getting a bit confusing with the introduction of many separate but similar sounding/seeming modules